recipes/rodeo/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "0.4.4" %}
+
+package:
+  name: rodeo
+  version: {{ version }}
+
+source:
+  fn: rodeo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/r/rodeo/rodeo-{{ version }}.tar.gz
+  md5: cbd93b992b508eb6f0915214b3201661
+
+build:
+  number: 0
+  entry_points:
+    - rodeo = rodeo.cli:cmd
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - jupyter
+    - flask >=0.10.1
+    - docopt
+    - pyzmq >=13
+    - mistune
+    - gnureadline  # [osx]
+
+  run:
+    - python
+    - jupyter
+    - flask >=0.10.1
+    - docopt
+    - pyzmq >=13
+    - mistune
+    - gnureadline  # [osx]
```

Comment:
[Appears]( https://travis-ci.org/conda-forge/staged-recipes/jobs/142480808#L2167-L2175 ) `rodeo` requires `pip`. Could we please add it? Expect it should pass after that change.